samples/Gallery/Pages/NotificationsPage.fs

```diff
@@ -93,6 +136,7 @@ module NotificationsPage =
 
     let view () =
         Component(program) {
+            //TODO What is this about?
```

Comment:
We have added support for Mvu Components and now the `model` is not part of the `view` function instead you can access it thought `let! model = Mvu.State` like you will do in the `async` CE. We are planning to add some docs at some point once this is more stable for now this is in preview

Reply:
I have created a separate repo `Fabulous.Avalonia.Samples` where I'm going create more small minimal samples ie  https://github.com/fabulous-dev/Fabulous.Avalonia.Samples/blob/main/MVU/CounterApp/App.fs

Reply:
Thanks! I'll wait for those docs and will have a look at the samples.